README.md

```diff
@@ -248,22 +248,59 @@ Benchmark comparison test [details data](https://gist.github.com/membphis/137db9
 
 A wide variety of companies and organizations use APISIX for research, production and commercial product, below are some of them:
 
+- Advanced Institute of Information Technology Peking University 
```

Comment:
Apache APISIX has a lot of users and this list will be very long. I suggest only listing some representative ones and then adding a link to the user page.

Reply:
Hi @bin-ya, before doing such changes, we'd better open an issue or Discussion first, after discussions and we're going to do it, we could submit a PR :)